src/Protocol/Models/VersionedTextDocumentIdentifier.cs

```diff
@@ -4,43 +4,15 @@
 namespace OmniSharp.Extensions.LanguageServer.Protocol.Models
 {
     [DebuggerDisplay("{" + nameof(DebuggerDisplay) + ",nq}")]
-    public class VersionedTextDocumentIdentifier : TextDocumentIdentifier, IEquatable<VersionedTextDocumentIdentifier>
+    public record VersionedTextDocumentIdentifier(DocumentUri Uri, int Version = 0) : TextDocumentIdentifier(Uri)
```

Comment:
There seem to be some behavior changes slipping in here (Version is now int instead of int?, defaults to 0 instead of null). That's chill if they were considered and intended, but personally I try to separate taking advantage of new features and making code-surface/behavior changes to avoid unexpected side-effects.

Reply:
Yeah this will be a breaking change, honestly it probably never should have been nullable, but that's how it is for the moment.